meson.build

```diff
@@ -814,6 +814,54 @@ if add_languages('cpp', required: llvmopt, native: false)
     # Some distros put LLVM and clang in different paths, so fallback to
     # find via PATH, too.
     clang = find_program(llvm_binpath / 'clang', 'clang', required: true)
+    llvm_lto = find_program(llvm_binpath / 'llvm-lto', required: true)
+    irlink = find_program('src/tools/irlink', native: true)
+
+    # Define a few bits and pieces used here and elsewhere to generate bitcode
+
+    llvm_irgen_args = [
+      '-c', '-o', '@OUTPUT@', '@INPUT@',
+      '-flto=thin', '-emit-llvm',
+      '-MD', '-MQ', '@OUTPUT@', '-MF', '@DEPFILE@',
+      '-O2',
+      '-Wno-ignored-attributes',
+      '-Wno-empty-body',
+    ]
+
+    if ccache.found()
+      llvm_irgen_command = ccache
+      llvm_irgen_args = [clang.path()] + llvm_irgen_args
+    else
+      llvm_irgen_command = clang
+    endif
+
+    llvm_irlink_kw = {
+      'command': [
+        irlink,
+        '--name', 'postgres',
+        '--lto', llvm_lto,
+        '--outdir', '@OUTPUT0@',
+        '--privdir', '@PRIVATE_DIR@',
+        '@INPUT@',
+      ],
+      'install': true,
+      'install_dir': dir_lib_pkg,
+    }
+
+    # XXX: Need to determine proper version of the function cflags for clang
+    bitcode_cflags = ['-fno-strict-aliasing', '-fwrapv']
+    bitcode_cflags += get_option('c_args')
+    bitcode_cflags += cppflags
+
+    # XXX: Worth improving on the logic to find directories here
```

Comment:
Still todo?

Reply:
I think that all of these should be investigated so we udnerstand why he added these and ideally we should fix all of them.